index.js

```diff
@@ -6,6 +6,8 @@ var os = require('os');
 var path = require('path');
 var resolve = require('resolve').sync;
 var spawn = require('child_process').spawn;
+var glob = require("glob");
+var async = require("async");
```

Comment:
seems like a funny name for it now. Is the library worth having the dependency now, rather than just a for loop?